modules/es-extensions/publisher/asset/policy/themes/default/js/view-asset.js

```diff
@@ -46,7 +46,8 @@ $(function() {
     });
 });
 $("#diff-view-version").on('change',function(){
-    var diff_view_url = "/publisher/pages/diff?type=wsdl&path=" + $("#diff-view-version").val() + ',' +
+    var assetType = store.publisher.type;
+    var diff_view_url = "/publisher/pages/diff?type="+assetType+"&path=" + $("#diff-view-version").val() + ',' +
         $("#diff-view-version").find(':selected').data('base_path');
```

Comment:
Thanks for pointing out the mistake, I have fix the similar issues in all the relevant parts.